addons/mail/static/src/js/wysiwyg.js

```diff
@@ -0,0 +1,97 @@
+/** @odoo-module **/
+
+import { renderToElement } from "@web/core/utils/render";
+import { Wysiwyg } from "@web_editor/js/wysiwyg/wysiwyg";
+import { closestBlock, setCursorEnd } from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
+import { patch } from "@web/core/utils/patch";
+import { MentionList } from "@mail/core/web/mention_list";
+import { url } from "@web/core/utils/urls";
+import { usePopover } from "@web/core/popover/popover_hook";
+
+patch(Wysiwyg.prototype, {
+    setup() {
+        super.setup();
+        if (this.props.options.recordInfo?.res_model === "mail.compose.message") {
+            this.mentionList = usePopover(MentionList, {
+                position: `bottom-start`,
+                onClose: () => {
+                    this.focus();
+                },
+            });
+        }
+    },
+    async startEdition() {
+        const res = await super.startEdition(...arguments);
+        // Only enable mention list in full chatter mode web editor
+        if (this.options.recordInfo?.res_model === "mail.compose.message") {
+            this.odooEditor.document.addEventListener(
+                "keydown",
+                this.triggerMentionList.bind(this),
+                true
+            );
+        }
+        return res;
+    },
+    destroy() {
+        super.destroy();
+        if (this.options.recordInfo?.res_model === "mail.compose.message" && this.odooEditor) {
+            this.odooEditor.document.removeEventListener(
+                "keydown",
+                this.triggerMentionList.bind(this),
+                true
+            );
```

Comment:
Should bind `triggerMentionList` in setup, so the function is bound and is same function between `on` and `off`

Reply:
We have the binding in `startEdition` since the binding needs info from `this.odooEditor` and this is the common case where the other event listeners are binded